install.sh

```diff
@@ -7,46 +7,47 @@ export GAMESCOPE_VERSION=3.11.52.beta2.r0.g2a09fbc-1
 pushd ~ >/dev/null
 if ! command -v "rwfus" &>/dev/null; then
     echo "Installing rwfus."
-    git clone https://github.com/ValShaped/rwfus.git >/dev/null
+    git clone -q https://github.com/ValShaped/rwfus.git
     pushd rwfus >/dev/null
     sudo ./rwfus -iI
     popd >/dev/null
-    rm -rf ./rwfus >/dev/null
+    rm -rf ./rwfus
 fi
-echo "Extracting gamescope update..."
+echo "Extracting gamescope update ${GAMESCOPE_VERSION}...\n"
 mkdir gamescope
 pushd gamescope
-wget https://builds.garudalinux.org/repos/chaotic-aur/x86_64/gamescope-git-$GAMESCOPE_VERSION-x86_64.pkg.tar.zst >/dev/null
-sudo tar --use-compress-program=unzstd -xvf ./gamescope-git-$GAMESCOPE_VERSION-x86_64.pkg.tar.zst >/dev/null
-echo
-echo "Installing gamescope update..."
-sudo rsync -a ./usr /opt/rwfus/mount/upper/ >/dev/null
-echo
-echo "Creating uninstall script..."
-sudo sh -c 'echo "#!/bin/sh" > /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "echo Uninstalling HDRDeck..." >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "pushd /opt/rwfus/mount/upper/" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'find ./usr -exec echo sudo rm -r {} + >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "sudo rm ./usr/bin/gamescope-session" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "sudo rm ./usr/bin/hdrdeck_uninstall" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "popd" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "read -p \"Please press enter to reboot.\"" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
-sudo sh -c 'echo "sudo reboot" >> /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall'
+wget -q https://builds.garudalinux.org/repos/chaotic-aur/x86_64/gamescope-git-${GAMESCOPE_VERSION}-x86_64.pkg.tar.zst
+sudo tar --use-compress-program=unzstd -xf ./gamescope-git-${GAMESCOPE_VERSION}-x86_64.pkg.tar.zst
+echo "Installing gamescope update...\n"
+sudo rsync -qa ./usr /opt/rwfus/mount/upper/
+echo "Creating uninstall script...\n"
+
+cat << EOF | sudo tee /opt/rwfus/mount/upper/usr/bin/hdrdeck_uninstall > /dev/null
+  #!/bin/sh
+  echo "Uninstalling HDRDeck..."
+  sudo rm -rv /opt/rwfus/mount/upper/usr
```

Comment:
This is a very bad idea, many people use Rwfus for other things so just removing the whole usr folder would break people's other stuff, not just uninstalling HDRDeck